data/map planets.txt

```diff
@@ -773,6 +794,14 @@ planet Clark
 	description `	The low cost of living and virtually zero unemployment rate make poverty and crime relatively rare occurrences. Instead there is a pioneering spirit, a startup-world attitude that anything is attainable with a bit of hard work and frugality.`
 	spaceport `The spaceport is at the heart of one of Clark's largest cities, surrounded by smokestacks and factories. There are a few hangars owned by different corporations where their cargo ships can load and unload, but you are relegated to a set of concrete landing pads farther from the main buildings. Large trucks rumble by carrying carts loaded down with cargo crates.`
 	spaceport `	In the central building, dozens of workers fill a large cafeteria with several stations serving different types of food. The mood here feels boisterous and energetic, very different from most of the Dirt Belt worlds you have visited.`
+		to display
+			or
+				"days since year start" <= 35
+				"days since year start" >= 55
+	spaceport `	In the central building, dozens of workers fill a large cafeteria with several stations serving different types of food. The mood here feels boisterous and energetic, bolstered further by Remembrance Day celebrations. Locals and tourists take turns downing as many beers as possible as their friends shout Alpha Wars shanties.`
```

Comment:
```suggestion
	spaceport `	In the central building, dozens of workers fill a large cafeteria with several stations serving different types of food. The mood here feels boisterous and energetic, bolstered further by Remembrance Day celebrations. Locals and tourists take turns downing as many beers as possible as their friends shout Alpha Wars anthems.`
```

Shanties are used in a few places for the descriptions, but those are work songs, typically sung on ships to keep time doing a task like rowing. I don't think it really works to describe folk songs that people sing to remember an event. Depending on what you're going for, you could use 

- Ballads as recounting a story, often emotionally and a bit more somber.
- Laments or hymns for an even more solemn and perhaps more religious feel.
- Anthems as a more boisterous and uplifting celebratory song, which I think makes the most sense for the drunk locals/tourists here.